src/components/ImageView/index.js

```diff
@@ -85,9 +86,17 @@ class ImageView extends PureComponent {
             imgRight = imgLeft + (fitRate * width);
         }
 
-        this.setState({
-            imgWidth: width, imgHeight: height, imageLeft: imgLeft, imageTop: imgTop, imageRight: imgRight, imageBottom: imgBottom,
-        });
+        // In case image loading is delayed than onLayout callback of the root View caused internet speed.
```

Comment:
I didn't really understand what this comment is about?

Reply:
> 

For regression, there is new issue that is open.
Because I didn't know which issue number should be written, so I made it.
Which issue number should be written?
Original one?

Reply:
For this comment, I noticed that image loading is delayed because of internet speed.
In this case, image size is not decided even though the container (modal) is open.
Because of this reason, `zoomScale` is also not decided.
To avoid this problem, I wrote the code and comment like above.

Reply:
> onLayout callback of the root View caused internet speed.

this is not making sense to me.

Please mention both issue URLs in the description.

Reply:
> 

It means 
1: You don't need such exception handling? Need to remove that code?
2. Should not write as `Original issue`, `Regression issue`?
Instead of it, should write urls directly like below?
`https://github.com/Expensify/App/issues/6326`, `https://github.com/Expensify/App/issues/6518`

Reply:
Hi, @parasharrajat 
Could you please let me know what you need exactly?

Reply:
I will let others review it. I don't really know this part of code.

Reply:
If you want, I can remove that code shortly.
But I added that code because I noticed the issue in my side.
Please let me know whenever you have any feedback.

Reply:
Hi, @parasharrajat , @deetergp 
How are you doing?
Any feedback from other reviewers?
Please kindly let me know so that we can proceed.
Thank you

Reply:
I tried hard to understand what you were trying to communicate with this comment, but it doesn't make sense to me. It sounds like there's some race condition at play here? If you can explain what you mean in a different way, that would be great. Otherwise, please just remove this comment.